linera-web/src/lib.rs

```diff
@@ -35,18 +35,19 @@ use crate::signer::JsSigner;
 
 // TODO(#12): convert to IndexedDbStore once we refactor Context
 type WebStorage =
-    linera_storage::DbStorage<linera_views::memory::MemoryStore, linera_storage::WallClock>;
+    linera_storage::DbStorage<linera_views::memory::MemoryDatabase, linera_storage::WallClock>;
 
 type WebEnvironment =
     linera_core::environment::Impl<WebStorage, linera_rpc::node_provider::NodeProvider, JsSigner>;
 
 type JsResult<T> = Result<T, JsError>;
 
-async fn get_storage() -> Result<WebStorage, <linera_views::memory::MemoryStore as WithError>::Error>
-{
+async fn get_storage(
+) -> Result<WebStorage, <linera_views::memory::MemoryDatabase as WithError>::Error> {
```

Comment:
Here the code is not a generic one. So, why use a complex definition and not `MemoryStoreError`?

Reply:
not my code